tests/lib/util/glob-util.js

```diff
@@ -29,7 +30,7 @@ var fixtureDir;
  */
 function getFixturePath() {
     var args = Array.prototype.slice.call(arguments);
-    args.unshift(fixtureDir);
```

Comment:
Those tests aren't failing because they already use `fs.realpathSync` inside the tests where needed. Maybe it could be refactored there instead, though.